admin_guide/seccomp.adoc

```diff
@@ -0,0 +1,127 @@
+[[admin-guide-seccomp]]
+= Restricting Application Capabilities Using Seccomp
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+[IMPORTANT]
+====
+Seccomp is
+link:https://github.com/kubernetes/kubernetes/blob/release-1.4/docs/design/seccomp.md[currently
+in alpha support].
+====
+
+Seccomp is used to restrict the set of system calls applications can make,
+allowing administrators greater control over the security of workloads running
+in {product-title}.
+
+Seccomp support is achieved via two annotations in the pod configuration:
+
+* *seccomp.security.alpha.kubernetes.io/pod*: profile applies to all containers in the pod that do not override
+* *container.seccomp.security.alpha.kubernetes.io/<container name>*: container-specific profile override
+
+[IMPORTANT]
+====
+Containers are run with *unconfined* seccomp settings by default.
+====
+
+For detailed design information, refer to the
+link:https://github.com/kubernetes/kubernetes/blob/release-1.4/docs/design/seccomp.md[seccomp
+design document].
+
+[NOTE]
+====
+Seccomp is a feature of the Linux kernel. To ensure seccomp is enabled on your
+system run:
+
+----
+$ cat /boot/config-`uname -r` | grep CONFIG_SECCOMP=
+CONFIG_SECCOMP=y
+----
+====
+
+[[seccomp-configuring-openshift-for-seccomp]]
+== Configuring {product-title} for Seccomp
+
+. Create the seccomp profile. 
++
+A seccomp profile is a json file providing syscalls and the appropriate action
+to take when a syscall is invoked. The
+link:https://github.com/docker/docker/blob/master/profiles/seccomp/default.json[default
+profile] is sufficient in many cases, the cluster administrator must define the
+security constraints of an individual system.
++
+To create your own custom profile, create a file on every node in the
+`seccomp-profile-root` directory.
++
+If you are using the default *runtime/default* profile, you do not need to
+create one.
+
+. Configure your nodes to use the *seccomp-profile-root* where your profiles
+will be stored. In the *node-config.yaml* via the *kubeletArguments*:
++
+----
+kubeletArguments:
+  seccomp-profile-root:
+    - "/your/path"
+----
+
+. Restart the node host.
+
+. In order to control which profiles may be used and to set the default profile,
+xref:../architecture/additional_concepts/authorization.adoc#authorization-seccomp[configure
+your SCC] via the *seccompProfiles* field. The first profile will be used as a
+default.
++
+When configuring seccomp profiles to use, follow these formatting rules:
+
+* *runtime/default*: the default profile for the container runtime (no profile on disk required)
+* *unconfined*: unconfined profile, ie, no seccomp sandboxing
+* *localhost/<profile-name>*: the profile installed to the node's local seccomp profile root
```

Comment:
I'm not sure what the above is saying. Are they example profiles? can they be substituted into the step above this one?